bundles/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/handler/IPBridgeThingHandler.java

```diff
@@ -171,12 +171,13 @@ public void initializeLater() {
         if (!config.getLocalIp().isEmpty()) {
             localEndPoint = new InetSocketAddress(config.getLocalIp(), 0);
         } else {
-            if (networkAddressService == null) {
+            final var localNetworkAddressService = networkAddressService;
```

Comment:
```suggestion
            NetworkAddressService localNetworkAddressService = networkAddressService;
```

Local variables don't need to be final because they can't be accessed by other threads anyway. We try to avoid using `var` because it hides the original type and is harder to read.

Reply:
There were more occurrences of "final var" in the binding, I removed all of them now....